tailwind.config.js

```diff
@@ -23,7 +28,7 @@ module.exports = {
       'default-border': '#dcdee1',
 
       // primary
-      'primary': '#2572b4',
+      'primary': '#293749',
       'primary-border': '#091c2d',
       'primary-hover': '#1c578a;',
 
```

Comment:
These updates became apparent when I started using the values on Team 2. There seems to be a bunch of different areas to retrieve styling from so I'm just using the colours/screens/font sizes that are available in designs that the DECD designers have instead of the WET/GC values.